apollo-biz/src/test/resources/application.properties

```diff
@@ -17,7 +17,7 @@ spring.cloud.consul.enabled=false
 spring.cloud.zookeeper.enabled=false
 spring.cloud.discovery.enabled=false
 
-spring.datasource.url = jdbc:h2:mem:~/apolloconfigdb;mode=mysql;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1
+spring.datasource.url = jdbc:h2:mem:~/apolloconfigdb;mode=mysql;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1;BUILTIN_ALIAS_OVERRIDE=TRUE
```

Comment:
What's the meaning of `BUILTIN_ALIAS_OVERRIDE=TRUE`? Should it also be configured in apollo-configservice/src/test/resources/application.properties and apollo-adminservice/src/test/resources/application.properties?

Reply:
H2 2.x has builtin function UNIX_TIMESTAMP. **CREATE ALIAS IF NOT EXISTS UNIX_TIMESTAMP** will need this